async_downloads/ws_consumers.py

```diff
@@ -55,21 +68,24 @@ async def receive(self, text_data):
         event = data["eventType"]
         if event == "clearDownload":
             await self.clear_download(data["data"]["filepath"])
-        if event == "initializeAllDownloads":
-            await self.initialize_all_downloads()
+        if event == "initAllDownloads":
+            await self.init_all_downloads()
 
-    async def initialize_all_downloads(self):
+    async def init_all_downloads(self):
         from async_downloads.cache import get_collection_key
 
         download_keys = cache.get(get_collection_key(self.user), [])
         downloads = []
-        for download_key in download_keys:
+        #  by default they are listed from newest
+        for download_key in reversed(download_keys):
             dl = cache.get(download_key)
             if not dl:
                 continue
             if dl["complete"]:
                 dl["url"] = default_storage.url(dl["filepath"])
+            dl["downloadKey"] = download_key
             dl["timestamp"] = str(dl["timestamp"])
+            dl["div_html"] = render_to_string(DOWNLOAD_TEMPLATE, {"downloads": [dl]})
```

Comment:
Minor thing, but I think just `html` is nicer instead of `div_html`?

Reply:
05bab57